### ISSUE TITLE
mavenn.run_demo error

### ISSUE THREAD

**Issue by mahdikooshkbaghi** [jbkinney/mavenn#12]
Here is the error running the `run_demo` command in `mavenn 0.22`
some lib versions:
`Python     3.7.4 `
`GCC     7.3.0`
`pandas     1.0.3`                  
`numpy     1.17.4`
`matplotlib                    3.3.2                  `
`tensorflow                    2.3.1                  `
`tensorflow-estimator          2.3.0             `     
`tensorflow-probability        0.10.1`

```python
mavenn.run_demo("gb1_ge_evaluation", print_code=False)
```

```python
Running /home/mahdik/.local/lib/python3.7/site-packages/mavenn/examples/demos/gb1_ge_evaluation.py...
Using mavenn at: /home/mahdik/.local/lib/python3.7/site-packages/mavenn
Model loaded from these files:
	/home/mahdik/.local/lib/python3.7/site-packages/mavenn/examples/models/gb1_ge_additive.pickle
	/home/mahdik/.local/lib/python3.7/site-packages/mavenn/examples/models/gb1_ge_additive.h5
---------------------------------------------------------------------------
RecursionError                            Traceback (most recent call last)
<ipython-input-27-d9f2f5bbcdbb> in <module>
----> 1 mavenn.run_demo("gb1_ge_evaluation", print_code=False)

~/.local/lib/python3.7/site-packages/mavenn/src/error_handling.py in wrapped_func(*args, **kwargs)
     96         try:
     97             # Execute function
---> 98             result = func(*args, **kwargs)
     99 
    100             # If running functional test and expect to fail

~/.local/lib/python3.7/site-packages/mavenn/src/examples.py in run_demo(name, print_code, print_names)
     97 
     98         # Run run_demo
---> 99         exec(open(file_name).read())
    100 
    101         print('Done!')

~/.local/lib/python3.7/site-packages/mavenn/src/examples.py in <module>

~/.local/lib/python3.7/site-packages/mavenn/src/error_handling.py in wrapped_func(*args, **kwargs)
     96         try:
     97             # Execute function
---> 98             result = func(*args, **kwargs)
     99 
    100             # If running functional test and expect to fail

~/.local/lib/python3.7/site-packages/mavenn/src/model.py in get_theta(self, gauge, p_lc, x_wt, unobserved_value)
    884 
    885         # Create unobserved_lc
--> 886         unobserved_lc = (x_stats['probability_df'].values == 0)
    887 
    888         # Set p_lc

~/apps/intelpython3/lib/python3.7/site-packages/pandas/core/generic.py in __getattr__(self, name)
   5270             return object.__getattribute__(self, name)
   5271         else:
-> 5272             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5273                 return self[name]
   5274             return object.__getattribute__(self, name)

~/apps/intelpython3/lib/python3.7/site-packages/pandas/core/generic.py in __getattr__(self, name)
   5270             return object.__getattribute__(self, name)
   5271         else:
-> 5272             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5273                 return self[name]
   5274             return object.__getattribute__(self, name)

... last 2 frames repeated, from the frame below ...

~/apps/intelpython3/lib/python3.7/site-packages/pandas/core/generic.py in __getattr__(self, name)
   5270             return object.__getattribute__(self, name)
   5271         else:
-> 5272             if self._info_axis._can_hold_identifiers_and_holds_name(name):
   5273                 return self[name]
   5274             return object.__getattribute__(self, name)

RecursionError: maximum recursion depth exceeded while calling a Python object

```

**Comment by atareen** [comment]
Hi Mahdi, this is a known error to me caused by a pandas version. mavenn needs pandas version>=1.1.2, that I have scheduled for the next update. But there are currently many items undergoing development.

**Comment by atareen** [comment]
Hi @mahdikooshkbaghi , I'll released version 0.23 of mavenn on PyPI which addresses this issue, can you update your mavenn version and retry the demo?

**Comment by mahdikooshkbaghi** [comment]
Hi @atareen, all ok with the `mavenn 0.23`. Thanks.
